compiler-rt/lib/scudo/standalone/primary64.h

```diff
@@ -251,9 +238,10 @@ template <typename Config> class SizeClassAllocator64 {
         }
 
         const bool RegionIsExhausted = Region->Exhausted;
-        if (!RegionIsExhausted)
+        if (!RegionIsExhausted) {
```

Comment:
Since this is a single line if, should it be done without braces?

That tends to be the current code style.

Reply:
Sounds good to me.